modules/oe_theme_content_consultation/src/Plugin/ExtraField/Display/ConsultationClosedTextExtraField.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_consultation\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\extra_field\Plugin\ExtraFieldDisplayFormattedBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Display Consultation closed status text.
+ *
+ * @ExtraFieldDisplay(
+ *   id = "oe_consultation_closed_status_text",
+ *   label = @Translation("Closed status text"),
+ *   bundles = {
+ *     "node.oe_consultation",
+ *   },
+ *   visible = true
+ * )
+ */
+class ConsultationClosedTextExtraField extends ExtraFieldDisplayFormattedBase implements ContainerFactoryPluginInterface {
+
+  use StringTranslationTrait;
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * EventExtraFieldBase constructor.
+   *
+   * @param array $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin_id for the plugin instance.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, EntityTypeManagerInterface $entity_type_manager) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->entityTypeManager = $entity_type_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('entity_type.manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getLabel() {
+    return $this->t('Closed status text');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(ContentEntityInterface $entity) {
+    $closed_text_field = '';
+    if (!$entity->get('oe_consultation_closed_text')->isEmpty()) {
+      $closed_text_field = $entity->get('oe_consultation_closed_text')->first()->getValue();
+    }
+
+    return [
+      '#theme' => 'oe_theme_content_consultation_closed_text',
```

Comment:
Can we use 
```
return ['#markup' => $text];
```
and field--node--extra-field-oe-consultation-closed-status-text.html.twig template to wrap text by .ecl-editor instead of additional template oe_theme_content_consultation_closed_text?